core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -76,6 +79,28 @@
    * <code>avro.compatibility.level</code>
    */
   public static final String COMPATIBILITY_CONFIG = "avro.compatibility.level";
+  public static final String KAFKASTORE_SECURITY_PROTOCOL_CONFIG =
```

Comment:
I'll experiment with @Ishiihara's suggestion below, to change `ConfigDef` to support null defaults. Regardless, I'll add a TODO per @hachikuji's suggestion.